<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsDLWindowDockWidgetBase</class>
 <widget class="QgsDockWidget" name="QgsDLWindowDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map Canvas</string>
  </property>
  <widget class="QWidget" name="mContents">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QToolBar" name="mToolbar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="floatable">
       <bool>false</bool>
      </property>
      <addaction name="action_tiqudianlixian"/>
      <addaction name="separator"/>
      <addaction name="action_polygonselectiononprofile"/>
      <addaction name="m_selectiononprofile"/>
      <addaction name="m_multipolygonselectiononprofile"/>
      <addaction name="mActionPickPoints"/>
      <addaction name="mActionSaveEdits"/>
      <addaction name="separator"/>
      <addaction name="action_chongjiandianlixian"/>
      <addaction name="separator"/>
      <addaction name="mActionPan"/>
      <addaction name="mActionMeasureAngle"/>
      <addaction name="mActionMeasureDistance"/>
      <addaction name="mActionTurnLeft"/>
      <addaction name="mActionTurnRight"/>
      <addaction name="actionmActionUpCenter"/>
      <addaction name="actionmActionDownCenter"/>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
   </layout>
  </widget>
  <action name="mActionPickPoints">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/propertyicons/render.svg</normaloff>:/images/themes/default/propertyicons/render.svg</iconset>
   </property>
   <property name="text">
    <string>拾取点，并显示在表格中</string>
   </property>
   <property name="toolTip">
    <string>拾取点，并显示在表格中</string>
   </property>
  </action>
  <action name="mActionSaveEdits">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionSaveAllEdits.svg</normaloff>:/images/themes/default/mActionSaveAllEdits.svg</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
  </action>
  <action name="mActionMeasureAngle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionMeasureAngle.svg</normaloff>:/images/themes/default/mActionMeasureAngle.svg</iconset>
   </property>
   <property name="text">
    <string>Measure Angle</string>
   </property>
  </action>
  <action name="mActionMeasureDistance">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionMeasure.svg</normaloff>:/images/themes/default/mActionMeasure.svg</iconset>
   </property>
   <property name="text">
    <string>测量距离</string>
   </property>
   <property name="toolTip">
    <string>MeasureDistance</string>
   </property>
  </action>
  <action name="mActionTurnLeft">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionArrowLeft.svg</normaloff>:/images/themes/default/mActionArrowLeft.svg</iconset>
   </property>
   <property name="text">
    <string>左旋</string>
   </property>
  </action>
  <action name="mActionTurnRight">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionArrowRight.svg</normaloff>:/images/themes/default/mActionArrowRight.svg</iconset>
   </property>
   <property name="text">
    <string>右旋转</string>
   </property>
  </action>
  <action name="mActionPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionPan.svg</normaloff>:/images/themes/default/mActionPan.svg</iconset>
   </property>
   <property name="text">
    <string>Pan Map</string>
   </property>
  </action>
  <action name="actionmActionUpCenter">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionArrowUp.svg</normaloff>:/images/themes/default/mActionArrowUp.svg</iconset>
   </property>
   <property name="text">
    <string>mActionUpCenter</string>
   </property>
  </action>
  <action name="actionmActionDownCenter">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionArrowDown.svg</normaloff>:/images/themes/default/mActionArrowDown.svg</iconset>
   </property>
   <property name="text">
    <string>mActionDownCenter</string>
   </property>
   <property name="toolTip">
    <string>mActionDownCenter</string>
   </property>
  </action>
  <action name="action_tiqudianlixian">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/cc/mapIcon.png</normaloff>:/images/cc/mapIcon.png</iconset>
   </property>
   <property name="text">
    <string>电力线拟合</string>
   </property>
  </action>
  <action name="action_chongjiandianlixian">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/algorithms/mAlgorithmFuzzifyGaussian.svg</normaloff>:/images/themes/default/algorithms/mAlgorithmFuzzifyGaussian.svg</iconset>
   </property>
   <property name="text">
    <string>电力线重建</string>
   </property>
  </action>
  <action name="action_polygonselectiononprofile">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/north_arrows/gpsarrow2.svg</normaloff>:/images/north_arrows/gpsarrow2.svg</iconset>
   </property>
   <property name="text">
    <string>多边形框选</string>
   </property>
  </action>
  <action name="m_selectiononprofile">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/cc/square.png</normaloff>:/images/cc/square.png</iconset>
   </property>
   <property name="text">
    <string>在剖面图上框选点云</string>
   </property>
  </action>
  <action name="m_multipolygonselectiononprofile">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/tips/symbol_levels.png</normaloff>:/images/tips/symbol_levels.png</iconset>
   </property>
   <property name="text">
    <string>多个多边形在剖面图上框选点云</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDockWidget</class>
   <extends>QDockWidget</extends>
   <header>qgsdockwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
